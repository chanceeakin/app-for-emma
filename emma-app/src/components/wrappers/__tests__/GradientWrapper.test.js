import React from 'react'
import Gradient from '../GradientWrapper'

import renderer from 'react-test-renderer'

describe('<GradientWrapper />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Gradient />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
