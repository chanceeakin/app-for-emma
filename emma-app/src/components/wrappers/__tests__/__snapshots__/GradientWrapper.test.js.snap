// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GradientWrapper /> renders correctly 1`] = `
<ExponentLinearGradient
  colors={
    Array [
      4287217054,
      4293262581,
    ]
  }
  endPoint={
    Array [
      1,
      1,
    ]
  }
  location={
    Array [
      0.25,
      0.4,
      1,
    ]
  }
  locations={null}
  startPoint={
    Array [
      0,
      0,
    ]
  }
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#f9f9f9",
      "flex": 1,
      "justifyContent": "space-between",
    }
  }
/>
`;
